person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(name = 'Unknown', age = 0, parent_permission: true)
+    @id = rand(1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
Just as required in this milestone, the `of_age` method is meant to be a private method😉. Kindly do well to implement this👍.
<img width="594" alt="image" src="https://github.com/naji-97/OOP-school-library/assets/64643995/edf238f5-3045-43b6-a80a-882da48cde0f">

```suggestion

  def can_use_services?
    of_age? || @parent_permission
  end
  
  private
  
  def of_age?
    @age >= 18
  end

```